libs/core/src/global/styles/tokens/$themes.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
Would you like us to introduce a theme here? I believe this will also give us the greatest flexibility with the Rebrand work.

Reply:
I don't think that'll be necessary since we'll be getting a mapping of old -> new tokens. With the way Tokens Studio works with theming, it allows you to toggle on certain sets of tokens to achieve that particular theme. I feel like theming will come more into play once we establish a set of semantic tokens.